content/en/docs/instrumentation/js/instrumentation.md

```diff
@@ -510,6 +512,198 @@ const doWork = (parent, i) => {
+* Asynchronous Counter, a asynchronous instrument which supports non-negative
+  increments
+* Histogram, a synchronous instrument which supports arbitrary values that are
+  statistically meaningful, such as histograms, summaries or percentile
+* Asynchronous Gauge, an asynchronous instrument which supports non-additive
+  values, such as room temperature
+* UpDownCounter, a synchronous instrument which supports increments and
+  decrements, such as number of active requests
+* Asynchronous UpDownCounter, an asynchronous instrument which supports
+  increments and decrements
+
+If a `MeterProvider` is not created either by an instrumentation library or
+manually, the OpenTelemetry APIs for tracing will use a no-op implementation and
+fail to generate data.
+
+### Initialize Metrics
+
+To initialize metrics, make sure you have the right packages installed:
+
+```shell
+npm install \
+  @opentelemetry/api \
+  @opentelemetry/resources \
+  @opentelemetry/semantic-conventions \
+  @opentelemetry/sdk-metrics \
+  @opentelemetry/instrumentation
+```
+
+Next, create a separate `instrumentation.js` file that has all the SDK
+initialization code in it:
+
+```js
+const otel = require('@opentelemetry/api')
+const {
+    MeterProvider,
+    PeriodicExportingMetricReader,
+    ConsoleMetricExporter
+  } = require('@opentelemetry/sdk-metrics');
+const { Resource } = require('@opentelemetry/resources');
+const { SemanticResourceAttributes } = require('@opentelemetry/semantic-conventions');
+
+const resource =
+  Resource.default().merge(
+    new Resource({
+      [SemanticResourceAttributes.SERVICE_NAME]: "service-name-here",
+      [SemanticResourceAttributes.SERVICE_VERSION]: "0.1.0",
+    })
+  );
+
+const metricReader = new PeriodicExportingMetricReader({
+    exporter: new ConsoleMetricExporter(),
+
+    // Default is 60000ms (60 seconds). Set to 1 second for demonstrative purposes only.
+    exportIntervalMillis: 3000,
+});
+
+const myServiceMeterProvider = new MeterProvider({
+  resource: resource,
+});
+
+myServiceMeterProvider.addMetricReader(metricReader);
+
+// Set this MeterProvider to be global to the app being instrumented.
+otel.metrics.setGlobalMeterProvider(myServiceMeterProvider)
+```
+
+Now that a `MeterProvider` is configured, you can acquire a `Meter`.
+
+### Acuiring a Meter
+
+Anywhere in your application where you have manual instrume code should call
+`getMeter` to acquire a meter. For example:
+
+```js
+const otel = require('@opentelemetry/api')
+
+const myMeter = otel.metrics.getMeter(
+  'my-service-meter'
+);
+
+// You can now use a 'meter' to create instruments!
+```
+
+It’s generally recommended to call `getMeter` in your app when you need it
+rather than exporting the meter instance to the rest of your app. This helps
+avoid trickier application load issues when other required dependencies are
+involved.
+
+### Synchronous and asynchronous instruments
+
+HELP
```

Comment:
Not sure if this helps. It is my uderstanding and a very simplistic description and leaves out the more interesting (but also more complicated) features of asynchronous instruments, such as multi-instrument callbacks. 

```suggestion
OpenTelemetry provides two major categories of instruments: synchronous and anynchronous (observable) instruments. Synchronous instruments can be used anywhere in the code. Using these, measurements can be pushed to the instrument at any time during the execution. This means that during an export cycle, multiple or no measurements can take place. Asynchronous instruments, on the other hand, provide a measurement at the request of the SDK. When the SDK exports, a callback that was provided to the instrument on creation is invoked. This callback provides the SDK with a measurement, which is then immediately exported. Therefore all measurement on asynchronous instruments are performed once per export cycle. 
```